.gitignore

```diff
@@ -0,0 +1,3 @@
+server/node_modules
+.env
+/server
```

Comment:
You might want to reconsider if you really want  to add `/server` to `gitignore`

Reply:
yes, it didn't eventually get ignored